ipa-core/src/protocol/ipa/mod.rs

```diff
@@ -1115,9 +1115,9 @@ pub mod tests {
                 cap_one(),
                 SemiHonest,
                 PerfMetrics {
-                    records_sent: 14_421,
-                    bytes_sent: 47_100,
-                    indexed_prss: 19_137,
+                    records_sent: 14_397,
```

Comment:
yay some minor gains :D

Reply:
It'll be more significant when using a larger number of breakdowns. This test is only using 4 breakdowns =). The savings is like ~O(num_breakdowns)